packages/providers/src.ts/base-provider.ts

```
@@ -1113,38 +728,8 @@ export class BaseProvider extends Provider implements EnsProvider {
         return this.formatter.filter(await resolveProperties(result));
     }
 
-    async call(transaction: Deferrable<TransactionRequest>, blockTag?: BlockTag | Promise<BlockTag>): Promise<string> {
-        await this.getNetwork();
-        const params = await resolveProperties({
-            transaction: this._getTransactionRequest(transaction),
-        });
-
-        const result = await this.perform("call", params);
-        try {
-            return hexlify(result);
-        } catch (error) {
-            return logger.throwError("bad result from backend", Logger.errors.SERVER_ERROR, {
-                method: "call",
-                params, result, error
-            });
-        }
-    }
-
     async estimateGas(transaction: Deferrable<TransactionRequest>): Promise<BigNumber> {
```

Instead of returning `0`, lets throw not implemented/not supported